core/build.gradle

```diff
@@ -42,6 +42,9 @@ dependencies {
     testImplementation "org.powermock:powermock-core:2.0.9"
     testImplementation "org.powermock:powermock-module-junit4:2.0.9"
     testImplementation "org.powermock:powermock-api-mockito2:2.0.9"
+
+    implementation "androidx.work:work-runtime:2.7.1"
```

Comment:
Make this "compileOnly"
If the developer wants this feature, they need to implement this.

Reply:
done